packages/editor/src/hooks/navigation-link-view-button.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * WordPress dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	__unstableBlockToolbarLastItem as BlockToolbarLastItem,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+// Target blocks that should have the View button
```

Comment:
This is a tiny nitpick, but I think we usually use full stops at the end of all comments. Although I guess we don't have a lint rule for this, so maybe it's not too important 👀 

```suggestion
// Target blocks that should have the View button.
```

There are a couple of other comments that may need full stops in this file too.